### DISCUSSION TITLE
Suggestion: label more issues as `good-first-issue`

### DISCUSSION THREAD

**Discussion by marie-bnl** [florisboard/florisboard#2355] [General]
This project is way too complex for me to contribute to its core. However I really like it and would love it to become a fully-featured keyboard. In order to be able to help achieving that, I would like more issues about tasks that don’t require to understand the codebase (for instance adding a keyboard layout or changes in isolated components) to be labeled as `good-first-issue`, so I can at least bring minor contributions to the project.

**Comment by barcode-Tm** [comment]
In the latest beta version, you can no longer activate the toolbar in two rows
